src/space.js

```diff
@@ -1,6 +1,7 @@
 const KeyValueStore = require('./keyValueStore')
 const Thread = require('./thread')
 const { sha256Multihash, throwIfUndefined, throwIfNotEqualLenArrays } = require('./utils')
+const OrbitDBAddress = require('orbit-db/src/orbit-db-address')
```

Comment:
We should really try to get these into the exports of orbit-db!

Reply:
yeah ideally